site/content/reference/api.md

```diff
@@ -0,0 +1,1151 @@
+---
```

Comment:
Is there any way to see this page rendered? It doesn't seem to have deployed in Netlify (although I might just be looking in the wrong place 😅)

Reply:
seems like it didn't deploy after a force push, we have it now https://deploy-preview-1884--nginx-gateway-fabric.netlify.app/nginx-gateway-fabric/reference/api/

It doesn't look good 😅

Reply:
We should have an example of generated API documentation for other software: I'll inquire with the team what the caveats are.

Reply:
The NGINX One API ref guide just calls the JSON spec and redoc does the formatting auto-magically.

See: https://github.com/nginxinc/docs/blob/main/content/nginx-one/reference/api/nginx-one-api-reference.md

Reply:
The doc type is "redoc"

<img width="371" alt="image" src="https://github.com/nginxinc/nginx-gateway-fabric/assets/33876974/980f1083-a945-4f4d-b6db-92f249e4f9b8">

Reply:
@travisamartin it's a little bit different from the Open API Spec, we don't have a JSON.
It's something similar to https://gateway-api.sigs.k8s.io/reference/spec/

Reply:
@ADubhlaoich @travisamartin any chance you can help with this?

Reply:
@lucacome Some more investigation work is required on our end about if an existing template can be adapted for this.

Does the Gateway API follow a specific, known schema?

Reply:
I'm not sure, this is just an html page